### DISCUSSION TITLE
на каких условиях разрешаете использование вашего репо в декстопном приложении❓ Questions / Help / Support

### DISCUSSION THREAD

**Discussion by ProBuro** [snakers4/silero-models#73] [Q&A]
## ❓ Questions and Help

на каких условиях разрешаете использование вашего репо в декстопном приложении❓

**Comment by snakers4** [comment]
В соответствии с указанной  лицензией, а так конечно зависит от приложения

**Comment by ProBuro** [comment]
> В соответствии с указанной лицензией, а так конечно зависит от приложения
приложение для пк (windows), озвучивает текст и файл, проставляет базовые ударения(с доработкой), проигрывает файл
